index.html

```diff
@@ -118,6 +118,143 @@ <h2>
           </li>
         </ul>
       </section>
+      <section class='informative'>
+        <h2>
+          Examples
+        </h2>
+        <section>
+          <h2>
+            Locking to a specific orientation and unlocking
+          </h2>
+          <p>
+            In this example, clicking the "Lock" button makes a request to go
+            into fullscreen and then lock to the opposite orientation. If the
+            screen is not in default orientation, pressing the "Unlock" button
+            unlocks to its default orientation.
+          </p>
+          <p>
+            The developer console logs the change in orientation type and
+            angle.
+          </p>
+          <pre class='example html'>
+  &lt;script&gt;
+  function fullScreenCheck() {
+    if (document.fullscreenElement) return;
+    return document.documentElement.requestFullscreen();
+  }
+  
+  function updateDetails(lockButton) {
+    const buttonOrientation = getOppositeOrientation();
+    lockButton.textContent = `Lock to ${buttonOrientation}`;
+  }
+  
+  function getOppositeOrientation() {
+    const { type } = screen.orientation;
+    return type.startsWith("portrait") ? "landscape" : "portrait";
+  }
+  
+  async function rotate(lockButton) {
+    try {
+      await fullScreenCheck();
+    } catch (err) {
+      console.error(err);
+    }
+    const newOrientation = getOppositeOrientation();
+    await screen.orientation.lock(newOrientation);
+    updateDetails(lockButton);
+  }
+  
+  function show() {
+    const { type, angle } = screen.orientation;
+    console.log(`Orientation type is ${type} & angle is ${angle}.`);
+  }
+  
+  screen.orientation.addEventListener("change", () => {
+    show();
+    updateDetails(document.getElementById("button"));
+  });
+  
+  window.addEventListener("load", () => {
+    show();
+    updateDetails(document.getElementById("button"));
+  });
+  &lt;/script&gt;
+  
+  &lt;button onclick="rotate() id="button"&gt;
```

Comment:
Note that this was still buggy... missing closing quote and we should pass in this.  

```suggestion
  &lt;button onclick="rotate(this)" id="button"&gt;
```